### DISCUSSION TITLE
How to get all tags that are assigned to a specific model type?

### DISCUSSION THREAD

**Discussion by Restingo** [spatie/laravel-tags#421] [Q&A]
How to get all tags that are assigned to a specific model type? Something like `Model::allTags()`? I want to use them for autocomplete function but scoped to models.

**Comment by Sindoweb** [comment]
Do you mean Model::getTags() ?

I added this to my Model:

```
public function getTags()
{
        $tags = [];
        foreach ($this->tags()->get() as $tag) {
            $tags[] = $tag->name;
        }

        return implode(', ', $tags);
}
```
